packages/frontend/src/components/buy/BuyNear.js

```diff
@@ -225,11 +226,13 @@ export function BuyNear({ match, location, history }) {
                     subTitle='buyNear.nearPurchaseSubTitle'
                     actions={[PayMethods.nearPay, PayMethods.moonPay, PayMethods.utorg, PayMethods.ftx]}
                 />
-                <FundingCard
-                    title='buyNear.bridgeTokens'
-                    subTitle='buyNear.bridgeSubTitle'
-                    actions={[PayMethods.rainbow]}
-                />
+                {!isWhitelabel && (
```

Comment:
```suggestion
                {!isWhitelabel() && (
```

Reply:
Not a blocker: may be worth to make `isWhiteLabel` a selector vs running this manually every time

Reply:
Oh! Thank you! Always forgot that this is a function. Actually naming like boolean variable really confusing. If we go deeper then why does it's even a function while it behaves like a real boolean flag!? No need to make any selector I refactor code to use it like a variable.